examples/faq/are_you_there.py

```diff
@@ -0,0 +1,65 @@
+from confluent_kafka import Producer, KafkaError
+
+
+class ConfluentException(Exception):
```

Comment:
There is a KafkaException already.

Reply:
I purposefully avoided using it since this is not raised by the underlying client. I wanted to avoid any confusion about this being something handled by confluent-kafka-python because this is an imperfect solution to a complicated problem.